src/snapshots/some_folder/002.txt

```diff
@@ -17,10 +17,10 @@ KwModule,OpenSquare,LowerIdent,Comma,LowerIdent,CloseSquare,Newline,LowerIdent,O
         (exposed_item (1:14-1:17) (lower_ident "bar")))
     (decl (3:1-3:4)
         (ident (3:1-3:4) "foo")
-        (string (3:7-3:8) "one"))
+        (string (3:7-3:12) (string_part (3:8-3:11) "one")))
     (decl (5:1-5:12)
         (ident (5:1-5:4) "bar")
-        (string (5:7-5:12) "two")))
+        (string (5:7-5:12) (string_part (5:8-5:11) "two"))))
```

Comment:
I wonder why one decl parsed with end at column 4 `decl (3:1-3:4)` and the other at column 12 `decl (5:1-5:12)`... curious